src/engines/html5/html5.js

```diff
@@ -904,6 +904,18 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @return {number[]} - playback rates
    */
   get playbackRates(): Array<number> {
+    if (Array.isArray(this._config.playback.playbackRate)) {
```

Comment:
this means every time the API is called we will have to do this check. Sounds a bit redundant.
Maybe add a setter and do the validation there? @yairans WDYT?

Reply:
agree